src/Connectors/src/Connectors/MongoDb/MongoDbWebApplicationBuilderExtensions.cs

```diff
@@ -2,20 +2,42 @@
 // The .NET Foundation licenses this file to you under the Apache 2.0 License.
 // See the LICENSE file in the project root for more information.
 
-#nullable enable
-
 using Microsoft.AspNetCore.Builder;
 using Steeltoe.Common;
 
 namespace Steeltoe.Connectors.MongoDb;
 
 public static class MongoDbWebApplicationBuilderExtensions
 {
+    /// <summary>
+    /// Registers a <see cref="ConnectorFactory{TOptions,TConnection}" /> to connect to a MongoDB database.
```

Comment:
Would it be more user-friendly if these comments specified the implementation types they're setting up instead of showing the generics?

Reply:
I tried that, but the values get discarded by the compiler, so they won't show up in IntelliSense. The only thing I can think of is to spell them out literally, for example:
```
Registers a <see cref="ConnectorFactory{TOptions,TConnection}" />
with type parameters <see cref="MongoDbOptions"/> and MongoDB.Driver.MongoClient
to connect to...
```
Which I think would be helpful. What do you think?

Reply:
That seems like it could be useful if Steeltoe users will need to use those types. I'm not currently deep enough into what it's like using these new connectors to have a solid opinion yet, so I'll leave it to your discretion as to whether it's worth the effort to add those

Reply:
Reopening to address this.

Reply:
It now looks like this:

![image](https://github.com/SteeltoeOSS/Steeltoe/assets/104792814/ff39fd37-d116-4f14-b43a-f13af429fdb8)